### PR TITLE
Additional Badges for Readme

### PR DESCRIPTION
**Pull request by confused-Techie**
This adds a few new badges to the readme, that I think we have been missing.

* Codacy
* Crowdin
* OpenCollective
* Reddit
* GitHub Discussions

It also removes the Logo Guidelines, as those were the Guidelines of GitHub and not ourselves (as suggested by @Daeraxa )

This also provides links for all of them obviously, and additionally spaces them out a little bit more, to ideally have seperate rows of badges for each 'subject'

- Install - Documentation - Building -

- License - Build Status - Upstream Status - 

- Codacy - Crowdin - OpenCollective -

- GitHub Discussion - Discord - Reddit -

But one important thing to note here, is currently all of our Badges use `for-the-badge` theming, but the badges from our providers don't follow this format.

Namely Codacy, Crowdin, and Open Collective.

Which those look like the following:

[Codacy Badge](https://www.codacy.com/gh/pulsar-edit/pulsar/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=pulsar-edit/pulsar&amp;utm_campaign=Badge_Grade)

[Crowdin](https://crowdin.pulsar-edit.dev/project/pulsar-edit)

[OpenCollective](https://opencollective.com/pulsar-edit)

Do we mind this style change? Do we want to convert all of our other badges to match this style?

### CODE REVIEW

**Comment by Daeraxa** [comment]
I do prefer the existing badge style but if it isn't supported then it isn't supported, I think this is the style that most projects tend to use now anyway.

The breakdown of the badges also makes a lot more sense I think although maybe Codacy should be with the CI build badge?

Do we even need or want to keep that logo guidelines badge?

**Comment by confused-Techie** [comment]
@Daeraxa great point about the logo guidelines. It links to GitHub's Guidelines. Not even ours at all, I'll remove it.

But exactly it's not supported, but if we like the 'For the Badge' then having some odd ducks doesn't seem to be a problem. Although personally I do like the more modern style myself. (What Codacy and such are using)

And I could move it, but it felt more appropriate here, as it's more of a build tool, rather than truly a CI thing, since it doesn't make any actual changes.

File: README.md [hunk 1]
```diff
@@ -58,17 +66,19 @@
...
-[Badge Guidelines]: https://img.shields.io/badge/Logo-Guidelines-d36e2d.svg?style=for-the-badge&labelColor=323232
 [Badge Retired]: https://img.shields.io/badge/Retired-bb3c1f.svg?style=for-the-badge&labelColor=323232&label=Upstream%20Status
 [Badge Sunset]: https://img.shields.io/badge/Sunset-orange.svg?style=for-the-badge&labelColor=323232&label=Upstream%20Status
 [Badge Discord]: https://img.shields.io/badge/Discord-6399c4.svg?style=for-the-badge&labelColor=323232&logoColor=white&logo=Discord
 [Badge License]: https://img.shields.io/badge/License-MIT-e5ab42.svg?style=for-the-badge&labelColor=323232
 [Badge Status]: https://img.shields.io/cirrus/github/pulsar-edit/pulsar?style=for-the-badge&labelColor=323232&label=Build%20Status
+[Badge Codacy]: https://app.codacy.com/project/badge/Grade/24873ecb93dc4c1d865202ce5b24efc1
+[Badge Crowdin]: https://badges.crowdin.net/pulsar-edit/localized.svg
+[Badge Reddit]: https://img.shields.io/reddit/subreddit-subscribers/pulsaredit?style=for-the-badge
+[Badge OpenCollective]: https://opencollective.com/pulsar-edit/tiers/badge.svg
```
Comment (Daeraxa) [inline README.md:78]:
```suggestion
[Badge Discussions]: https://img.shields.io/github/discussions/pulsar-edit/.github?style=for-the-badge
```

File: README.md [hunk 2]
```diff
@@ -34,10 +39,13 @@
...
 <!---------------------------{ Links }--------------------------->
 
-[Guidelines]: https://github.com/logos 'Branding Guidelines'
 [Electron]: https://github.com/electron/electron
 [Discord]: https://discord.gg/7aEbB9dGRT 'Join the Pulsar Discord today!'
 [Status]: https://cirrus-ci.com/github/pulsar-edit/pulsar/master
+[Codacy]: https://app.codacy.com/gh/pulsar-edit/pulsar
+[Crowdin]: https://crowdin.pulsar-edit.dev
+[Reddit]: https://www.reddit.com/r/pulsaredit/
+[OpenCollective]: https://opencollective.com/pulsar-edit
```
Comment (Daeraxa) [inline README.md:49]:
```suggestion
[Discussions]: https://github.com/orgs/pulsar-edit/discussions
```

File: README.md [hunk 3]
```diff
@@ -11,14 +11,19 @@
...
+[![Badge License]][License] 
 [![Badge Status]][Status]    
 [![Badge Sunset]][Retired]    
+
+[![Badge Codacy]][Codacy]
+[![Badge Crowdin]][Crowdin]
+[![Badge OpenCollective]][OpenCollective]
+
 [![Badge Discord]][Discord]
+[![Badge Reddit]][Reddit]
```
Comment (Daeraxa) [inline README.md:23]:
```suggestion
[![Badge Discussions]][Discussions]
[![Badge Discord]][Discord]
[![Badge Reddit]][Reddit]
```

**Review by Daeraxa — COMMENTED** [review]
Just had a thought to add in the GH discussions whilst the conversation about badges is open...

**Comment by confused-Techie** [comment]
> Just had a thought to add in the GH discussions whilst the conversation about badges is open...

Great Idea, committed your suggestions directly

**Comment by confused-Techie** [comment]
But with this does anyone from @pulsar-edit/documentation want to take a look?

File: README.md [hunk 4]
```diff
@@ -11,14 +11,20 @@
 [![Button Documentation]][Documentation]   
 [![Button Build]][Build] 
 
-[![Badge License]][License]    
-[![Badge Guidelines]][Guidelines]    
+[![Badge License]][License] 
 [![Badge Status]][Status]    
 [![Badge Sunset]][Retired]
```
Comment (mauricioszabo) [inline README.md:16]: Can we remove this sunset badge?

**Comment by confused-Techie** [comment]
@mauricioszabo I've gone ahead and removed that sunset badging and links, which makes sense, as we shouldn't be reporting the status of upstream as our own

**Review by autumnblazey — APPROVED** [review]

**Comment by confused-Techie** [comment]
With approval from @autumnblazey I'll go ahead and merge this one. Any additional changes can of course be made later on. But would be nice to see us get to a low amount of PRs.

Thanks everyone to all the help on this one.

**Pull request merged**